modules/ROOT/pages/_partials/data-sync/server-offline-and-conflict.adoc

```diff
@@ -1,45 +1,24 @@
-
-In a GraphQL server, conflict detection and resolution happens exclusively in *mutations*.
-
-== Resolving Conflicts
-
-At a high level, there is a typical flow to detecting and resolving conflicts.
+At a high level, there is a typical flow to detecting conflicts.
 
 . *A Mutation Occurs* - A client tries to modify or delete an object on the server using a GraphQL mutation.
 . *Read the Object* - The server reads the current object the client is trying to modify from the data source (usually a database).
 . *Conflict Detection* - The server compares the current object with the data sent by the client to see if there was a conflict. The developer can choose _how the comparison_ is done.
-. *Conflict Resolution* - Conflict resolution can be deferred to the client or it can be done on the server. The developer can choose the resolution strategy.
-. *Persist the Data* - Conflict Resolution on the server results in a new object which should be persisted.
-
-The `aerogear/voyager-conflicts` module uses the concept of *pluggable conflict resolution* to help developers with the *Conflict Detection* and *Conflict Resolution* steps.
-
-== Pluggable Conflict Resolution
-
-Pluggable conflict resolution is a concept that allows developers to define their own logic for *conflict detection* and *conflict resolution*, regardless of the data storage.
-
-It has two parts: one for detecting conflicts, and the other for resolving conflicts.
 
-To detect conflicts, developers can use either the default version-based conflict detection mechanism, or provide their own implementation via the `conflictStateProvider` option in the config object that is used to initialize the sync client.
+== Pluggable Conflict Detection
+The `aerogear/voyager-conflicts` module uses the concept of *pluggable conflict detection* to help developers with the *Conflict Detection* steps regardless of their storage.
 
-To resolve conflicts, developers can either use the default conflict resolution strategy, or provide their own ones via the `conflictStrategy` option in the config object.
+The conflict detection is enabled by {sync-server}, while the fetching and storing of data is the responsibility of the developer.
 
-The conflict detection and resolution is enabled by {sync-server}, while the fetching and storing of data is the responsibility of the developer.
+Pluggable conflict detection supports the following implementations:
 
-Pluggable conflict resolution supports the following implementations:
+* `VersionedObjectState` - depends on version field supplied in objects (used by default when importing conflictHandler). For details, please see: xref:version-based-conflict[]
+* `HashObjectState` - depends on hash calculated from the entire object. For details, please see: xref:hash-based-conflict[]
```

Comment:
* `HashObjectState` - depending on the hash calculated from the entire object. For details, please see: xref:hash-based-conflict[]